vpn-router-alice/src/main/java/vpnrouter/alice/security/AliceAuthorizer.java

```diff
@@ -0,0 +1,27 @@
+package vpnrouter.alice.security;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import vpnrouter.alice.infrastructure.controller.AliceRequest;
+
+@Component
+@RequiredArgsConstructor
+public class AliceAuthorizer {
```

Comment:
I've read a little about authorization with [Alice](https://yandex.ru/dev/dialogs/alice/doc/ru/auth/how-it-works). To be honest, some details remained unclear to me. How does it correspond to the article "Как происходит авторизация в навык" or is it something else?

Reply:
No, I don't authorize a user in our system. I just check the IDs of a request because I know what skill (I created it), user (me) and application (Yandex Station) I expect.